jaeger-core/src/main/java/com/uber/jaeger/reporters/protocols/JaegerThriftSpanConverter.java

```diff
@@ -48,7 +48,7 @@ private JaegerThriftSpanConverter() {}
             context.getTraceId(),
             0, // TraceIdHigh is currently not supported
             context.getSpanId(),
-            context.getParentId(),
+            span.getParentContext() == null ? 0 : span.getParentContext().getSpanId(),
```

Comment:
See the description of this PR. 

I would like to use `parentId` in thrift span only if there is one parent (`childOf`). In all other cases use a list of references. You are right I can set `parentId` to null if references are null (see changed code).